auth/src/AzureSubscriptionProvider.ts

```diff
@@ -19,7 +19,7 @@ export interface AzureSubscriptionProvider {
      *
      * @returns A list of tenants.
      */
-    getTenants(account?: vscode.AuthenticationSessionAccountInformation): Promise<TenantIdDescription[]>;
+    getTenants(account?: vscode.AuthenticationSessionAccountInformation): Promise<AzureTenant[]>;

```

Comment:
This is technically a breaking change, but it's harmless. The break would be a compile error to anyone implementing this interface. Bump to 4.0.0 or to 3.2.0?

Reply:
I'll add some more changes and figure it out later